recipes/looptools/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "looptools" %}
+{% set version = "2.16" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://feynarts.de/looptools/LoopTools-{{ version }}.tar.gz
+  sha256: 4113467575db3a14405d62d9e516b3b90410b73ea8d20c8eb8d70a30fc5cc9cb
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}-static
+
+    build:
+      skip: true  # [win]
+      run_exports:
+        - {{ pin_subpackage('looptools-static', max_pin='x.x') }}
+      ignore_run_exports_from:
+        - {{ compiler('cxx') }}
+      script:
+        # FIXME: Avoid error from trying to create directories that exist
+        - sed -i 's/mkdir /mkdir -p /g' makefile.in
+        - ./configure --help
+        # FIXME: Author has disabled builds with dynamic linking
+        # c.f. https://github.com/hep-packaging-coordination/packaging-hep-simulation-stack/issues/4#issuecomment-2489198926
+        # - ./configure --prefix=$PREFIX --dynamic --64 $CXXFLAGS
+        - ./configure --prefix=$PREFIX --64 $CXXFLAGS
```

Comment:
I've added `$CXXFLAGS` but kept the `--64`. I presume that's okay, though if that's something that's discouraged I can check to see how much LoopTools uses this `--64` information.

Reply:
Oh, it just does

```
	CONF_BITS="-m64" ;;
```

in the `configure`. This can get dropped.